src/storage/subscriptions.ts

```diff
@@ -0,0 +1,57 @@
+import { IPersistence, IPersistenceRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { RocketChatAssociationModel, RocketChatAssociationRecord } from '@rocket.chat/apps-engine/definition/metadata';
+import { ISubscription } from '../interfaces/external';
+
+export class Subscription {
+  constructor(private readonly persistence: IPersistence, private readonly persistenceRead: IPersistenceRead) {}
+
+  public async createSubscription(boardName: string, boardId: string, webhookId: string, rcUserId: string, miroUserId: string): Promise<boolean> {
+    try {
+      const associations: Array<RocketChatAssociationRecord> = [new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, `subscription`), new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, `webhookId:${webhookId}`), new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, `boardId:${boardId}`), new RocketChatAssociationRecord(RocketChatAssociationModel.USER, rcUserId)];
+      let subscriptionRecord: ISubscription = {
+        webhookId,
+        rcUserId,
+        boardName,
+        boardId,
+        miroUserId
+      };
+      await this.persistence.updateByAssociations(associations, subscriptionRecord, true);
+    } catch (error) {
+      console.warn('Subscription Error :', error);
```

Comment:
Instead of using the `console.log`, prefer using the `app.getLogger()`

Reply:
Fixed in next PR.